Assignment7.txt

```diff
@@ -0,0 +1,19 @@
+Clarke Littlejohn
+
+1) if i recall correctly ++*P moves the pointer then gets the values, *p++ get  the value in the point the increments it
+   and *++p is not an apoerator that works.
+
+2) I am confused by what this si asking it will go by what it is prendence so to will go from right to left or left to right based on the highest order
+
+3) you are working directly with the value so it is always modifing the number.
+
+4.1)string/char array/
+.2)invalid
+.3)invalid
+.4)pointer this gets the address of the array
+.5)address this gets the address
+.6)int this derefences the value
+.7)pointer address p is pointer and ampersand gets the address
```

Comment:
```right so the answer is int** for the a similar reason as number 5. The address of p returns a pointer to the address, but since p is of type int*, that means it will return the memory address of where the memory address is stored - in other words, a pointer to a pointer.```